types/stompit/index.d.ts

```diff
@@ -0,0 +1,21 @@
+// Type definitions for stompit 0.26
+// Project: https://github.com/gdaws/node-stomp
+// Definitions by: Daniel Rose <https://github.com/DanielRose>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types="node" />
+
+import { Writable, Readable, Duplex, WritableOptions } from "stream";
+import { EventEmitter } from "events";
+
+export import IncomingFrameStream = require("./lib/IncomingFrameStream");
+export import OutgoingFrameStream = require("./lib/OutgoingFrameStream");
```

Comment:
just a structural comment: it's a little weird to use commonjs syntax for submodules, but have the resulting module be an ES6 module. It might be more elegant to have submodules export their individual items instead of using `export = ...` and then use the syntax `export { IncomingFrameStream } from "./lib/IncomingFrameStream"`.

Reply:
The submodules are CommonJS, i.e. `module.exports = Channel;` and similar. The index.js is

```js
module.exports = {
   IncomingFrameStream:  require('./lib/IncomingFrameStream'),
  OutgoingFrameStream:  require('./lib/OutgoingFrameStream'),
  /...
};
```

How should I have written the type definition for that?